app/formats/document_types.yml

```diff
@@ -10,6 +10,11 @@
       type: govspeak
       validations:
         min_length: 10
+  associations:
```

Comment:
Do we need to have a 'fields' subhash? I noticed later on we write 'document_type_schema.associations', implying we're not expecting the contents/associations to have any other members.